src/openforms/contrib/digid_eherkenning/utils.py

```diff
@@ -17,3 +17,11 @@ def get_eherkenning_logo(request) -> dict[str, str]:
         "href": "https://www.eherkenning.nl/",
         "appearance": LogoAppearance.light,
     }
+
+
+def get_eidas_logo(request) -> dict[str, str]:
+    return {
```

Comment:
why is the coverage here missing? The other functions are covered: https://app.codecov.io/github/open-formulieren/open-forms/blob/master/src%2Fopenforms%2Fcontrib%2Fdigid_eherkenning%2Futils.py